package.json

```diff
@@ -47,7 +47,7 @@
     "mocha": "^6.2.2"
   },
   "dependencies": {
-    "bn.js": "^4.4.0",
+    "bn.js": "^5.0.0",
```

Comment:
@fanatid 
`5.1.1` is available.
https://github.com/indutny/bn.js/blob/master/CHANGELOG.md